packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -177,7 +170,7 @@ export const Timeline: React.FC<TimelineProps> = ({
       startDate={startDate}
       today={today}
     >
-      <TimelineToolbar />
+      <TimelineToolbar scaleOptions={scaleOptions} />
```

Comment:
I wonder if this should be composable? Maybe that's unnecessary.

I imagine consumers might want to supplement the toolbar with their own 'actions'.

Reply:
Yeah that is a good longer term plan 👍